index.html

```diff
@@ -254,26 +254,28 @@ <h2>Get to know us better</h2>
       </main>
     </div>
     <footer class="footer">
-      <ul class="menu">
-        <li class="menu__item">
-          <div class="social-icons">
-            <a href=""><span></span></a>
-            <a href=""><span></span></a>
-            <a href=""><span></span></a>
-            <a href=""><span></span></a>
-            <a href=""><span></span></a>
-          </div>
-        </li>
-        <li>
-          <div class="logo">
-            <img src="#" alt="logo" class="logo__img"/>
-            <p class="logo__name">Expire</p>
-          </div>
-        </li>
-        <li>
-          © Copyright 2014, design by dingo_design
-        </li>
-      </ul>
+      <div class="content">
+        <ul class="menu-footer">
+          <li class="menu-footer__item">
+            <div class="social-icons">
+              <a href=""><span></span></a>
+              <a href=""><span></span></a>
+              <a href=""><span></span></a>
+              <a href=""><span></span></a>
+              <a href=""><span></span></a>
+            </div>
+          </li>
+          <li>
+            <div class="logo">
+              <img src="#" alt="logo" class="logo__img"/>
+              <p class="logo__name">Expire</p>
+            </div>
+          </li>
+          <li>
+            © Copyright 2014, design by dingo_design
```

Comment:
For special characters, it would be better to use coding (© - `&copy;`)
https://www.w3schools.com/charsets/ref_html_entities_4.asp